peer.go

```diff
@@ -1236,6 +1240,94 @@ func (p *peer) logWireMessage(msg lnwire.Message, read bool) {
 	}))
 }
 
+// messageTypeString returns a human and monitoring-readable string that describes an
+// incoming/outgoing message type.
+func messageTypeString(msg lnwire.Message) string {
```

Comment:
The `String` method of `lnwire.Message` already does this. My guess is that this is in place due to naming conventions that prefer snake case for metrics? In that case, we method can easily be made to convert the camel case names to snake case. The bonus of this is that then we wouldn't need to maintain the mapping in two distinct areas within the codebase.

Reply:
No, camel case is also fine. I didn't realise it is a protobuf message with String method. I will make the code use the String method!

Reply:
Here's the method I'm talking about: https://github.com/lightningnetwork/lnd/blob/master/lnwire/message.go#L59